src/commands/pack/win.ts

```diff
@@ -74,7 +74,16 @@ Section "${config.name} CLI \${VERSION}"
   SetOutPath $INSTDIR
   File /r bin
   File /r client
-  ExpandEnvStrings $0 "%COMSPEC%"
+
+  ; Use explicit System32/Sysnative path to cmd.exe for security
+  StrCpy $0 "$WINDIR\\System32\\cmd.exe"  ; Try System32 first
+  IfFileExists "$0" path_is_safe
+    StrCpy $0 "$WINDIR\\Sysnative\\cmd.exe"  ; Try Sysnative for WOW64
+    IfFileExists "$0" path_is_safe
+      MessageBox MB_OK|MB_ICONSTOP "Error: Could not find system cmd.exe. Installation cannot continue."
+      Abort
+      
+  path_is_safe:
```

Comment:
Are you sure these are the paths for all Windows systems?  I'd hate to break customers that have a valid cmd.exe path that are not one of these two.